app/helpers/rad_common/bootstrap_flash_helper.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RadCommon
+  module BootstrapFlashHelper
+    ALERT_TYPES = %i[success info warning danger].freeze unless const_defined?(:ALERT_TYPES)
+
+    # rubocop:disable all
```

Comment:
Why are we disabling rubocop here?

Reply:
Just because this code was pulled from a battle-tested gem, and rubocop was complaining about a bunch of stuff that I figured we wouldn't care to fix right now